lib/iex/test/iex/interaction_test.exs

```diff
@@ -19,6 +19,31 @@ defmodule IEx.InteractionTest do
     assert capture_iex("1 + 2") == "3"
   end
 
+  if IO.ANSI.enabled?() do
```

Comment:
I don't think we need this check. We can always force it to be enabled using the `put_env` below.

Reply:
> I don't think we need this check. We can always force it to be enabled using the `put_env` below.

I remember in a previous PR you mentioned ansi tests would break in systems where ansi was not available, but if it's safe in this case we can remove it

Reply:
Oh, sorry for the confusion. Setting the app env overrides the systems settings, so it behaves independent on the OS. :)